docs/blog/2019-06-10-building-a-jobs-board-a-case-study/index.md

```diff
@@ -0,0 +1,40 @@
+---
+title: Building a jobs board with gatsby, a case study
+date: 2019-06-10
+author: Steven Natera
+tags:
+  - case-study
+---
+
+Today we are going to discuss how we built [Remotefordays](https://remotefordays.com), a jobs board for remote software engineer roles, with Gatsby. First we'll discuss how we made came to our tech stack, then we'll talk about the the gotchas, and falls starts we encountered along the way.
+
+# Components of the tech stack
+
+Gatsby is an amazing tool for rapid prototyping high performance websites on the frontend. We can get a new project started without much hassle due to a phenomenal developer experience. Therefore choosing Gatsby for our frontend concerns was easy. Next we needed to decide how to build and store job listings.
```

Comment:
@muescha remember to focus on substantive improvements! This is hazy at best as to whether it hits that mark!

Reply:
i am not a native speaker so i don't know what is the best. 

PS: i just stumbled over `frontend concern` so i had to google + translate it to find out it is not a framework nor a technical term for a middleware...

Reply:
@muescha that's valuable feedback then! My mistake--and thanks for noting this! We should keep the terms we use approachable, because in this case, they're effectively the same--if one has a clearer meaning, that's better!